code/Kotoba/UIViewController+Dictionary.swift

```diff
@@ -12,30 +12,8 @@ extension UIViewController
 	/// - returns: `true` if the system dictionary found a definition, `false` otherwise.
 	func showDefinition(forWord word: Word, completion: (() -> Void)? = nil)
 	{
-		let dictionaryViewController = UIReferenceLibraryViewController(term: word.text)
+		let dictionaryViewController = DictionaryViewController(term: word.text)
 		dictionaryViewController.modalPresentationStyle = .pageSheet
 		self.present(dictionaryViewController, animated: true, completion: completion)
-
-		// Prompt the user to set up their iOS dictionaries, the first time they use this only
-		if USER_PREFS.shouldDisplayFirstUseDictionaryPrompt()
```

Comment:
This code was moved to `DictionaryViewController` so that the SwiftUI view could still run it.